[Examples] vLLM example for SkyServe + Mixtral (#2948)
skypilot-org/skypilot: llm/vllm/README.md

  }
}
```

> ## Serving Mixtral 8x7b model with vLLM and SkyServe

Reviewer: We already have the mixtral 8x7b + vLLM and SkyServe in [llm/mixtral](llm/mixtral). Should we just make the example above to be launchable with sky serve and have an additional link refering to the [llm/mixtral](llm/mixtral)?

Author: Make sense! Changed. PTAL again 🫡

  }
}
```

## Serving the above Llama-2 example with vLLM and SkyServe

> 1. Adding an `service` section in the above `serve-openai-api.yaml` file to make it an [`SkyServe Service YAML` file](https://skypilot.readthedocs.io/en/latest/serving/service-yaml-spec.html):

Reviewer: ```suggestion
1. Adding an `service` section in the above `serve-openai-api.yaml` file to make it an [`SkyServe Service YAML`](https://skypilot.readthedocs.io/en/latest/serving/service-yaml-spec.html):
```

```yaml
# The newly-added `service` section to the `serve-openai-api.yaml` file.
service:
  # Specifying the path to the endpoint to check the readiness of the service.
  readiness_probe: /v1/models
  # How many replicas to manage.
  replicas: 2
```

> The whole Service YAML is shown here: [serve-openai-api-service.yaml](service.yaml).

Reviewer: ```suggestion
The entire Service YAML can be found here: [service.yaml](service.yaml).
```
